examples_src/overviewmap-custom.js

```diff
@@ -14,7 +14,7 @@ var overviewMapControl = new ol.control.OverviewMap({
   layers: [
     new ol.layer.Tile({
       source: new ol.source.OSM({
-        'url': '//{a-c}.tile.opencyclemap.org/cycle/{z}/{x}/{y}.png'
+        'url': 'https://{a-c}.tile.opencyclemap.org/cycle/{z}/{x}/{y}.png'
```

Comment:
It looks like it was a mistake to try to load via HTTPS here.  I'm getting `https://c.tile.opencyclemap.org/cycle/2/1/1.png` for https://c.tile.opencyclemap.org/cycle/2/1/1.png.  I'll make this HTTP.